enterprise/web.beans/test/unit/src/org/netbeans/modules/web/beans/model/InterceptorResolutionTest.java

```diff
@@ -138,7 +138,7 @@ public void testInterceptorNonbindingMembers() throws IOException{
                 "@Target({METHOD, TYPE}) "+
                 "public @interface IBinding2  {" +
                 " String value() ;"+
-                " @Nonbinding String comment() defualt \"\" ; "+
+                " @Nonbinding String comment() default \"\" ; "+
```

Comment:
Are you changing a test here?

Reply:
![image](https://user-images.githubusercontent.com/15821889/222924639-cce5d650-646a-456f-82f2-4c62b75331ea.png)
over bindings are equal. Is this correct?